pil-analyzer/src/evaluator.rs

```diff
@@ -976,8 +1005,36 @@ impl<'a, 'b, T: FieldElement, S: SymbolLookup<'a, T>> Evaluator<'a, 'b, T, S> {
             Expression::FreeInput(_, _) => Err(EvalError::Unsupported(
                 "Cannot evaluate free input.".to_string(),
             ))?,
-            Expression::StructExpression(_, _) => {
-                unimplemented!("Struct expressions are not yet supported.")
+            Expression::StructExpression(
+                _,
+                StructExpression {
+                    name: reference,
+                    fields,
+                },
+            ) => {
+                let mut exp_fields = Vec::new();
+                for NamedExpression { name, body } in fields.iter() {
```

Comment:
Consider changing `body` to `value` inside `NamedExpression` (if it makes sense in all cases)

Reply:
For now it's used only in Traits and Structs. Since they are NamedExpressions it seems to me that the most accurate would be `expr`.

What do you think?

Reply:
Yes, makes sense for `NamedExpression`  to use `name` and `expr` :)